test/puppetlabs/http/client/sync_ssl_test.clj

```diff
@@ -161,27 +161,6 @@
           (is (= 200 (:status response)))
           (is (= "Hello, World!" (slurp (:body response))))))))
 
-  (testing "should be able to connect to a server with non-default protocol if configured"
```

Comment:
Using SSLv3 in these test seems like an incidental part of the implementation in three of them and the fourth is validating that we _don't_ negotiate down to SSLv3 even if a server allows it. None of them seem like things that need to be removed for enabling fips support, though I could see some, if not all of them, being low value compared to maintenance costs. Am I missing something about that?

If their relative value was your rationale I didn't catch that from the commit message/can we add it?

Reply:
The behavior of these SSLv3 tests is radically different in a bc-fips env. The errors are different and everything.

Considering SSLv3 is disabled at the JRE level, I don't see any value in attempting to override those settings, so this felt almost "vestigial" for lack of a better term.